apps/jsonrpc2/lib/bridge/sync.ex

```diff
@@ -45,4 +46,30 @@ defmodule JSONRPC2.Bridge.Sync do
       _ -> nil
     end
   end
+
+  def get_transaction_by_block_hash_and_index(block_hash, trx_index) do
```

Comment:
Should these return the `Blockchain.Transaction` instead of `ResponseTransaction`? The caller can call `ResponseTransaction.new` if they wish to create one from it. The answer may be "no, they shouldn't. This works fine."

Reply:
these methods are used in `SpecHandler` so objects should be serializable with Jason. Our internal structs cannot be serialized with Jason